gtk4/src/font_dialog.rs

```diff
@@ -0,0 +1,111 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{FontDialog, Window};
+use glib::translate::*;
+use glib::IsA;
+use std::boxed::Box as Box_;
+use std::pin::Pin;
+use std::ptr;
+
+impl FontDialog {
+    #[doc(alias = "gtk_font_dialog_choose_font_and_features")]
+    #[doc(alias = "gtk_font_dialog_choose_font_and_features_finish")]
+    pub fn choose_font_and_features<
+        P: FnOnce(Result<(pango::FontDescription, glib::GString, pango::Language), glib::Error>)
```

Comment:
Is it really guaranteed that either all 3 are NULL and the error is set, or none of them are NULL?

Reply:
Right, FontDescription can be NULL, the two others should always be set if error is NULL.